### DISCUSSION TITLE
General Chat

### DISCUSSION THREAD

**Discussion by Denellyne** [Denellyne/PCXSense#6] [General]
General chat
In case you want to ask me something @Denellyne [Your question]!!

**Comment by IhateStevia** [comment]
@Denellyne 

Hello there!

Any future plans to support ps access controller?

Thanks!

**Reply by Denellyne** [reply]
What's a ps acess controller?

**Reply by IhateStevia** [reply]
https://www.playstation.com/en-id/accessories/access-controller

Sorry for the late reply

**Reply by Denellyne** [reply]
Assuming that uses the same Hid input/output of the dualsense then the only thing i would need to make that compatible would be the hid product id, so just to be sure,do you own one of those devices?

**Reply by Denellyne** [reply]
@IhateStevia Open this [website](https://hardwaretester.com/gamepad) and send me a screenshot like this,you need to connect the controller to your pc(the acess controller)
![image](https://github.com/Denellyne/PCXSense/assets/56112881/d580fafe-d45d-4d2f-8c5b-71ed4857036e)

**Reply by IhateStevia** [reply]
![image](https://github.com/Denellyne/PCXSense/assets/101363387/cbd6f989-a98f-4970-ad77-771b4e912a8e)
Only b1 b2 b9 b10 and axis 0/1 seem to be responding to input.

The rest are not bound and can only be assigned to input, by going through the setup on a PS5.

https://youtu.be/cYHdXMPC94g?si=ADHw4baqVRYpsXJw&t=150

**Reply by Denellyne** [reply]
Ok but here's the thing,my app makes the controller show as a Xbox controller,so i'm sorry but if the acess controller isnt working on pc to begin with then I really can't do much,maybe try setting it up on ps5 first

**Comment by FarisRaza1** [comment]
@Denellyne 
Couple of general questions:
1: when im connected to the controller through bluetooth and play a game (for example F123) it shows as an xbox controller rather than a ps5 controller. Is there any way to make it a ps5 controller. 
2: is there an option to minimize to tray rather than just have the application open all the time that way it just runs in the background without bein at the task bar. with that also is there an option to start at launch?

**Reply by Denellyne** [reply]
1 your not supposw to use my app with games compatible with the dualsense ,it shows as an xbox controller because thats what it does
2 im trying to get a solution for that believe me its hardee than it seems
Hope it answers your questions

**Reply by FarisRaza1** [reply]
@Denellyne 
yea it does thanks! i wanted to use your app so that I can connect my controller wirelessly rather than always having it plugged in but yea it just switches the inputs to xbox. Maybe something in the future can change that.

**Comment by FlpCastilho** [comment]
@Denellyne 
Hello! One question and one suggestion, if I may
For the question, is Linux support planned/possible?
As for the suggestion, maybe try including your software in [PCGamingWiki's DualSense Page](https://www.pcgamingwiki.com/wiki/Controller:DualSense#Windows).

Anyways, thanks for the great work, ever since DS4Windows has stopped development we've been lacking a good DualSense configuration tool!

**Reply by Denellyne** [reply]
I don't believe a linux build is out of scope but for now I'm focusing on the windows build because I'm not experienced with linux but I will probably get around to it as a means to learn Linux

**Comment by SamuelV2** [comment]
Do the haptics work? and if not, is there a future plan for them?

**Reply by Denellyne** [reply]
Haptic Feedback is a per game feature,I can't implement it sorry

**Comment by merlinch** [comment]
@Denellyne can this app can set the deadzone like on ds4windows?

**Reply by Denellyne** [reply]
No,i chose not to add that because games usually set their own deadzones

**Comment by drseussgaming** [comment]
My dual sense edge shows in the device manager but says disconnected in pcxsense.  Am I missing something?

**Reply by Denellyne** [reply]
The current release isn't compatible with the dualsense edge ,though the next will.

**Comment by AmeliaGL0w0** [comment]
i have a dualshock 4 connected via bluetooth, but this program doesnt detect it, yet the readme says the program supports ds4. do i need to use usb?
(i dont have a reliable micro usb with data transfer that i can use)

**Comment by Victtus** [comment]
Hi, the R2 and L2 buttons aren't showing up or being detected. My controller is a Dual Sense Edge. Is there any problem with setting this controller up that I'm not aware of? Thx

**Reply by Denellyne** [reply]
The dual sense Edge isnt supported given i dont own One ,Im sorry but theres nothing i can do to help

**Comment by itzzmirror** [comment]
@Denellyne hey, when i downloaded your software, there was a cmd flash i am really scared of trojans and malware because i had a trojan a  few months ago and almost lost access to everything i had. could you please explain why there was a console cmd flash when i opened your software?

**Reply by Denellyne** [reply]
It used to have an autoupdater but because i dont have a certificate ir usually got flagged as a vírus ,so i decides to remove the update festure but it still checks for a new version ar startup and if it finds one it tells u about it,maybe when i add support for Linux you Will see it

**Reply by itzzmirror** [reply]
so just to make sure the cmd flash is the program looking for a update? im on windows

**Reply by Denellyne** [reply]
Yes,exactly

**Comment by eVenent** [comment]
Hello, plans for gyro and touchpad support are dropped or we can expect new update in the future?

**Reply by Denellyne** [reply]
I plan to but im occupied rn

**Reply by eVenent** [reply]
Sure. Take your time. We will wait.

**Comment by Sicvoul** [comment]
Hi everyone. I wanna play skate. but skate doesnt see my bluetooth controller. I downloaded PCX, but all i got is 
<img width="1288" height="690" alt="image" src="https://github.com/user-attachments/assets/473d14ee-15bf-47b5-b438-6a61bcd61f50" />
What should i do?
